### PR TITLE
fix, --no-color is not work

### PR DESCRIPTION
**Pull request by xiangshouding**
detail #90

### CODE REVIEW

**Comment by Raynes** [comment]
I really, really need this. Please.

**Comment by backflip** [comment]
This would be really nice (if a plugin uses the prototype syntax I'd still like to be able to disable the colors using this flag).

**Comment by Raynes** [comment]
Aye, don't want to fork this guy, but really need this for a project I'm working on.

**Comment by 2betop** [comment]
I really, really need this. Please.

**Comment by zhang-ning** [comment]
＋1

**Comment by Raynes** [comment]
Or, at least explain why this isn't being merged. If it's a time thing, I can just fork it and move forward. If it's a code thing, I'm sure one of us can fix it. :)

**Comment by pksunkara** [comment]
Hello guys, I am going to review this on the weekend. Will update soon.

File: lib/colors.js [hunk 1]
```diff
@@ -48,6 +48,9 @@ colors.stripColors = colors.strip = function(str){
 
 
 var stylize = colors.stylize = function stylize (str, style) {
+  if (!colors.enabled) {
+    return str+'';
+  }
```
Comment (pksunkara) [inline lib/colors.js]: nitpick: Please add an empty line after this to make the code more readable.

**Pull request merged**